components/Chat/ChatDetailBody.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle, TextStyle } from 'react-native';
+import { AppTheme } from '../../config/DefaultConfig';
+import useTheme from "../../hooks/useTheme";
+import ThemedText from '../UI/ThemedText';
+import TimeDuration from './TimeDuration';
+
+interface Props {
+    message: string;
+    timeStamp: Date;
+    messageDirect: boolean;
+};
+
+const ChatDetailBody: React.FunctionComponent<Props> = ({
+    message,
+    timeStamp,
+    messageDirect,
+}: Props) => {
+    const theme: AppTheme = useTheme();
+
+    return (
+        <View style={[style.container, messageDirect ? {flexDirection: 'row'}: {flexDirection: 'row-reverse'}]}>
+            <View style={ messageDirect ? style.topContentContainer : style.flipContentContainer }>
+                <View style={style.timeContainer}>
+                    <TimeDuration
+                        date={timeStamp}
+                    />
+                </View>
+            </View>
+            <View style={ messageDirect ? [style.topContentData, {backgroundColor: theme.lightBottomColor}] :  [style.flipContentData, {backgroundColor: 'rgb(176, 224, 230)'}]}>
```

Comment:
convert `'rgb(176, 224, 230)'` to hex and define it in the theme file with the name `alternateMessageBackgroundColor` and then use it here

Reply:
I have changed it